### PR TITLE
Tools refactoring

### PR DESCRIPTION
**Pull request by simpletonDL**
I think all tools should be part of the same python script. So in terms of ```argparse``` we can use subparsers. Something like this: ```python3 init tool_1 [tool_1 args]```, ```python3 init tool_2 [tool_2 args]``` and so on. This will cause CFPQ_Data to be formed as a single package.

### CODE REVIEW

File: init.py [hunk 1]
```diff
@@ -1,339 +1,22 @@
...
-
-    ids = _idgen()
-    graph = []
-    sierpinski(1, 2, 3, degree, predicates, graph)
-    with open(
-        os.path.join(target_dir, 'sierpinskigraph_%s.txt' % (degree)), 'w'
-    ) as out_file:
-        for triple in graph:
-            out_file.write('%s %s %s \n' % (triple[0], triple[1], triple[2]))
+#!/usr/bin/python3
```
Comment (gsvgit) [inline init.py:1]: Seems, that script is not only for initialization from now. Should it be renamed?

File: src/utils/filters.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+def all_filter_combinator(*filters):
+    return reduce(lambda acc, f: lambda x: acc(x) and f(x), filters, lambda _: True)
+
+
+def file_has_extension(file, extensions=None):
+    return extensions is None or any([file.endswith(ext) for ext in extensions])
+
+
+def file_has_size(file, left_bound=None, right_bound=None):
```
Comment (gsvgit) [inline src/utils/filters.py:17]: Why file size, not stored graph size?

File: init.py (hunk 1 above)
Comment (simpletonDL) [reply init.py:1]: Yes, I wanted to rename it, but I didn't come up with anything. Can you suggest a better name?

File: src/utils/filters.py (hunk 2 above)
Comment (simpletonDL) [reply src/utils/filters.py:17]: I think there should be both options. There are problems with the size of graphs. Exactly to compute it for rdf files at first we need to parse this files. This takes a long time. So maybe the graph size should be approximated by the number of lines in the file?

Comment (gsvgit) [reply src/utils/filters.py:17]: Well. We plan to add metadata for graphs (number of vertices and edges). So, can we use this data here?

File: init.py (hunk 1 above)
Comment (gsvgit) [reply init.py:1]: No, I cannot. Maybe just ```main```.

File: src/utils/filters.py (hunk 2 above)
Comment (simpletonDL) [reply src/utils/filters.py:17]: Yes, of course. I can add additional flags and corresponding filters when the metadata appears.

**Pull request merged**